src/messages.rs

```diff
@@ -36,18 +36,18 @@ pub use alter_configs_request::AlterConfigsRequest;
 pub mod alter_configs_response;
 pub use alter_configs_response::AlterConfigsResponse;
 
-pub mod alter_isr_request;
-pub use alter_isr_request::AlterIsrRequest;
-
-pub mod alter_isr_response;
-pub use alter_isr_response::AlterIsrResponse;
-
 pub mod alter_partition_reassignments_request;
 pub use alter_partition_reassignments_request::AlterPartitionReassignmentsRequest;
 
 pub mod alter_partition_reassignments_response;
 pub use alter_partition_reassignments_response::AlterPartitionReassignmentsResponse;
 
+pub mod alter_partition_request;
+pub use alter_partition_request::AlterPartitionRequest;
+
+pub mod alter_partition_response;
+pub use alter_partition_response::AlterPartitionResponse;
+
```

Comment:
@tychedelia Here introduced a breaking change, since to the `AlterIsr` API has been renamed to `Alterpartition` in the current Kafka schema, in [ KAFKA-13587](https://github.com/apache/kafka/pull/11733)

Reply:
Reviewing the schema change, it looks like this isn't a breaking change at the protocol level, just the source code level in terms of names, is that correct?

Reply:
Sort of. The underlying API of the Kafka protocol not changed except it's name.

I mean, since the AlterIsrRequest name has changed, that is a breaking change to the crate users.

Reply:
Since we're pre-1.0 and this crate isn't in wide use yet, that totally seems fine to me. We might make a comment somewhere about building off latest (or a pinned release version as you suggested) so people can look to the protocol documentation for correct names, if they're used to using an older version.